app/src/main/java/org/p2p/wallet/infrastructure/network/NetworkModule.kt

```diff
@@ -103,4 +149,43 @@ object NetworkModule : InjectionModule {
             .addNetworkInterceptor(ContentTypeInterceptor())
             .build()
     }
+
+    private fun createCertificate(trustedCertificateIS: InputStream): SSLContext? {
```

Comment:
```suggestion
    private fun createCertificate(trustedCertificate: InputStream): SSLContext? {
```

you have IS in your type already